src/types.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="spotify-api" />
```

Comment:
It worked by adding the types and not use the reference

`npm i -sD @types/spotify-web-api-node`